node/blockchain/views/signed_change_request.py

```diff
@@ -1,15 +1,53 @@
+from rest_framework import status
+from rest_framework.exceptions import ValidationError as DRFValidationError
 from rest_framework.mixins import CreateModelMixin
+from rest_framework.response import Response
+from rest_framework.settings import api_settings
 from rest_framework.viewsets import GenericViewSet
 
-from node.blockchain.serializers.signed_change_request import SignedChangeRequestSerializer
+from node.blockchain.facade import BlockchainFacade
+from node.blockchain.models import Block
+from node.blockchain.serializers.signed_change_request import (
+    NodeDeclarationSignedChangeRequestSerializer, SignedChangeRequestSerializer
+)
+from node.core.exceptions import ValidationError
+from node.core.utils.cryptography import get_signing_key
+from node.core.utils.types import Type
 
 
 class SignedChangeRequestViewSet(CreateModelMixin, GenericViewSet):
     serializer_class = SignedChangeRequestSerializer
+    serializer_classes = {
+        Type.NODE_DECLARATION: NodeDeclarationSignedChangeRequestSerializer,
+    }
+
+    def get_serializer(self, *args, **kwargs):
```

Comment:
why do we need this method? is not type identification already handled in `SignedChangeRequestSerializer` ?

Reply:
Done.